cmd/kubeadm/app/phases/addons/dns/manifests.go

```diff
@@ -361,4 +361,74 @@ metadata:
   name: coredns
   namespace: kube-system
 `
+
+	// NodeLocalDNSCacheDaemonSet is the node local cache DaemonSet manifest
+	NodeLocalDNSCacheDaemonSet = `
+  apiVersion: apps/v1
+  kind: DaemonSet
+  metadata:
+    name: {{ .DeploymentName }}
+    namespace: kube-system
+    labels:
+      k8s-app: kube-dns
+      kubernetes.io/cluster-service: "true"
+      addonmanager.kubernetes.io/mode: Reconcile
```

Comment:
remove `kubernetes.io/cluster-service: "true"` and `addonmanager.kubernetes.io/mode: Reconcile`, that's GKE-specific stuff